sklearn/datasets/openml.py

```diff
@@ -489,26 +581,35 @@ def fetch_openml(name=None, version='active', data_id=None, data_home=None,
         If True, returns ``(data, target)`` instead of a Bunch object. See
         below for more information about the `data` and `target` objects.
 
+    as_frame : boolean, default=False
+        If True, returns a Bunch where the data attribute is a pandas
```

Comment:
This needs updating. Maybe just say "data in the returned bunch, or X in the returned (X, y), will be a data frame including columns appropriate dtypes (numeric, string or categorical)."